.github/policies/issue.flag-for-triage.yml

```diff
@@ -15,5 +16,37 @@ configuration:
       then:
       - addLabel:
           label: 'Needs: Triage (Functions)'
+
+    - description: Add needs attention label to reopened issues
+      if:
+      - payloadType: Issues
+      - and:
+        - isAction:
+            action: Reopened
+        - not:
+            hasLabel: 'Needs: Attention :wave:'
+      then:
+      - addLabel:
+          label: 'Needs: Attention :wave:'
+
+    - description: Assign issues when marked for triage or when attention needed again
+      if:
+      - payloadType: Issues
+      - isAction:
+          action: Labeled
+      - not:
+          isAssignedToSomeone
+      - or:
+        - labelAdded:
+            label: 'Needs: Triage (Functions)'
+        - labelAdded:
+            label: 'Needs: Attention :wave:'
+      then:
+        - assignIcmUsers:
+            teamId: 114785
+            primary: True
+            secondary: False
+      triggerOnOwnActions: true
+
```

Comment:
This is now part of the same file as the core assignment of those issues. This enforces ordering of the automations. If also added specific triggering on the label action. The goal is to ensure that the assignment action isn't evaluated prior to the necessary labels being added.